source/use-cases/sensitive-data-encryption.txt

```diff
@@ -203,6 +203,18 @@ Additional Dependencies
          * - `uuid-base64
              <https://www.npmjs.com/package/uuid-base64#installation>`_
            - Convert between Base64 and hexadecimal UUIDs.
+   .. tab::
+      :tabid: python
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Dependency Name
+           - Description
+
```

Comment:
This is more of a question for @terakilobyte and @nlarew -- since it appears from the ecosystem pages that we support two Python Drivers (PyMongo and Motor), should we specify that this uses pymongo only, and if so, where?